hig.js

```diff
@@ -82,7 +82,7 @@ function runSpeech() {
 	while (speak(out)) {}
 }
 
-function speak(phrases) {
+async function speak(phrases) {
```

Comment:
Why `async`? Is there an `await` inside?

Reply:
No, this was a mistake. My local changes are different. The code actually does not work when async is defining the function. I need to get rid of that. Thanks.

Reply:
Oh, this is outdated. I changed this back to a regular function. This was due to me trying to figure out what async actually did but I realized what it did and changed the function back. As you can see the updated file does not have async.